config/runtime/error_handler.go

```diff
@@ -13,8 +13,8 @@ import (
 )
 
 var wildcardMap = map[string][]string{
-	"api":      {"beta_insufficient_permissions"},
-	"endpoint": {"beta_insufficient_permissions", "sequence", "unexpected_status"},
+	"api":      {"backend", "backend_openapi_validation", "backend_timeout", "beta_insufficient_permissions"},
```

Comment:
Further down:
```go
		// this works if wildcard is the only "super kind"
		if mappedKinds, mkExists := wildcardMap[ref]; mkExists {
...
```
With this change, wildcard is not the only "super kind" any longer. So the mechanism used to expand the "super kinds" must now be modifed.